UnityProject/Assets/Scripts/ChemistryComponents/GAS2ReagentSingleton.cs

```diff
@@ -58,8 +59,10 @@ private void CheckState()
 			GasToReagent = new Dictionary<Gas, Reagent>();
 			foreach (var _ReagentToGas in InitialReagentToGas)
 			{
-				ReagentToGas[IntToReagent(_ReagentToGas.Key)] = _ReagentToGas.Value;
-				GasToReagent[_ReagentToGas.Value] = IntToReagent(_ReagentToGas.Key);
+					Debug.Log(IntToReagent(_ReagentToGas.Key));
+					Debug.Log(_ReagentToGas.Key);
```

Comment:
Either convert to Logger.Log() with a category if this is still needed or removed